plugins/user.ts

```diff
@@ -31,7 +31,7 @@ const userPlugin: Plugin = (context, inject) => {
   const user = {
     set: async (id: string, serverUrl: string, accessToken: string) => {
       const response = await context.$displayPreferencesApi.getDisplayPreferences(
-        { displayPreferencesId: 'usersettings', userId: id, client: 'emby' }
+        { displayPreferencesId: 'usersettings', userId: id, client: 'vue' }
```

Comment:
Would it be better to call this Jellyfin Vue?

Reply:
I think it's just a short identifier for the DB, it's not actually shown anywhere, afaik.

From what I understand, it allows you to have a different config for different clients.

Reply:
Ok, Sounds good to me